src/test/resources/fixtures/deadCode/parameters-writes-only.php

```diff
@@ -2,23 +2,31 @@
 
 class Container {
     public function method1(array $in, array &$out, $log) {
-        return
+        $outc = 1;
+        $function =
             function
                 (array $inner, array &$outer)
                 use (
                     <weak_warning descr="The variable seems to be not used.">$in</weak_warning>,
-                    &$out,
+                    &<weak_warning descr="The variable seems to be not used.">$out</weak_warning>,
```

Comment:
I have created more tests on this file to make sure that other cases are not generating false-positives (it help me on manual tests step).

`&$out` was declared on `use()` but not on parent context, and never used (then _warn_);
`&$outb` was declared on `use()`, but not on parent context, but was used (then _ok_);
`&$outc` was declared on `use()`, and too on parent context, and too used (then _ok_);